src/main/java/org/codewars/utils/TaskRunner.java

```diff
@@ -3,106 +3,116 @@
 import java.math.BigInteger;
 
 public class TaskRunner {
-    public static void setUser(int id){
+
+    private final Users user;
+
+    public static void setUser(int id) {
         // TODO: Implement setUser() method
     }
+
     public static long[] runGap() {
         return new long[0];
     }
 
-    public static BigInteger runPerimeter(){
+    public static BigInteger runPerimeter() {
         return new BigInteger("0");
     }
 
-    public static int runZeros(){
-       return 0;
+    public static int runZeros() {
+        return 0;
     }
 
-    public static double runF(){
+    public static double runF() {
         return 0.0;
     }
 
-    public static long runFindNb(){
+    public static long runFindNb() {
         return 0;
     }
 
-    public static double runMean(){
+    public static double runMean() {
         return 0;
     }
 
-    public static double runVariance(){
+    public static double runVariance() {
         return 0;
     }
 
-    public static String runBalance(){
+    public static String runBalance() {
         return null;
     }
 
-    public static String runNbaCup(){
+    public static String runNbaCup() {
         return null;
     }
 
-    public static String runStockSummary(){
+    public static String runStockSummary() {
         return null;
     }
 
-    public static double runGetVolumeOfCuboid(){
+    public static double runGetVolumeOfCuboid() {
         return 0.0;
     }
 
-    public static int runLiters(){
+    public static int runLiters() {
         return 0;
     }
 
-    public static int runWhereIsHe(){
+    public static int runWhereIsHe() {
         return 0;
     }
 
-    public  static long runNewAvg(){
+    public static long runNewAvg() {
         return 0;
     }
 
-    public static String runSeriesSum(){
+    public static String runSeriesSum() {
         return null;
     }
 
-    public static int runStringToNumber(){
+    public static int runStringToNumber() {
         return 0;
     }
 
-    public static int[] runcountPositivesSumNegatives(){
+    public static int[] runcountPositivesSumNegatives() {
         return new int[0];
     }
 
-    public static float runMpgToKPM(){
+    public static float runMpgToKPM() {
         return 0.0f;
     }
 
-    public static double runTwoDecimalPlaces(){
-        return 0.0;
+
+    public static void runTwoDecimalPlaces() {
```

Comment:
we have already decided we won't use static method in TaskRunner

Reply:
of course
done